src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusService.scala

```diff
@@ -115,8 +115,19 @@ class ConsignmentStatusService(
 }
 
 object ConsignmentStatusService {
-  val validConsignmentTypes: List[String] =
-    List("Series", "TransferAgreement", "Upload", "ClientChecks", "DraftMetadata", "ClosureMetadata", "DescriptiveMetadata", "ConfirmTransfer", "Export")
+  private val validConsignmentTypes: List[String] =
+    List(
+      "Series",
+      "TransferAgreement",
+      "Upload",
+      "ClientChecks",
+      "DraftMetadata",
+      "ClosureMetadata",
+      "DescriptiveMetadata",
+      "ConfirmTransfer",
+      "Export",
+      "MetadataReview"
```

Comment:
I believe the status is supposed to be `DTAReview`

Reply:
Actually, we call it now "Metadata Review" so I used it as `MetadataReview` type and it makes more sense to have as `MetadataReview` type because all the pages are called Metadata Review.